Library/Homebrew/cmd/install.rb

```diff
@@ -27,65 +119,25 @@ def install_args
-      switch "-s", "--build-from-source",
-             description: "Compile <formula> from source even if a bottle is provided. "\
-                          "Dependencies will still be installed from bottles if they are available."
-      switch "--force-bottle",
-             description: "Install from a bottle if it exists for the current or newest version of "\
-                          "macOS, even if it would not normally be used for installation."
-      switch "--include-test",
-             description: "Install testing dependencies required to run `brew test` <formula>."
-      switch "--HEAD",
-             description: "If <formula> defines it, install the HEAD version, aka. master, trunk, unstable."
-      switch "--fetch-HEAD",
-             description: "Fetch the upstream repository to detect if the HEAD installation of the "\
-                          "formula is outdated. Otherwise, the repository's HEAD will only be checked for "\
-                          "updates when a new stable or development version has been released."
-      switch "--keep-tmp",
-             description: "Retain the temporary files created during installation."
-      switch "--build-bottle",
-             description: "Prepare the formula for eventual bottling during installation, skipping any "\
-                          "post-install steps."
-      flag   "--bottle-arch=",
-             depends_on:  "--build-bottle",
-             description: "Optimise bottles for the specified architecture rather than the oldest "\
-                          "architecture supported by the version of macOS the bottles are built on."
       switch "-f", "--force",
-             description: "Install without checking for previously installed keg-only or "\
-                          "non-migrated versions."
+             description: "Install formulae without checking for previously installed keg-only or "\
+                          "non-migrated versions. Overwrite existing files when installing casks."
       switch "-v", "--verbose",
              description: "Print the verification and postinstall steps."
-      switch "--display-times",
-             env:         :display_install_times,
-             description: "Print install times for each formula at the end of the run."
-      switch "-i", "--interactive",
-             description: "Download and patch <formula>, then open a shell. This allows the user to "\
-                          "run `./configure --help` and otherwise determine how to turn the software "\
-                          "package into a Homebrew package."
-      switch "-g", "--git",
-             description: "Create a Git repository, useful for creating patches to the software."
+
       conflicts "--ignore-dependencies", "--only-dependencies"
-      conflicts "--devel", "--HEAD"
       conflicts "--build-from-source", "--build-bottle", "--force-bottle"
+
+      formula_only_options.each do |options|
```

Comment:
Could this be inlined in this function as `formula_only_options` isn't used elsewhere and it'd allow you to read through the list of options in order (as before)?

Reply:
I had this before but RuboCop complained that the block is too long and I didn't want to fight it.

Reply:
@reitermarkus Bah. I feel like ideally these usage blocks would be excluded automatically and, if not, we could exclude `cmd` and `dev-cmd` entirely